internal/app/repository/menu_repository.go

```diff
@@ -0,0 +1,128 @@
+package repository
+
+import (
+	"context"
+	"errors"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/opentracing/opentracing-go"
+	"github.com/voltgizerz/POS-restaurant/internal/app/entity"
+	"github.com/voltgizerz/POS-restaurant/internal/app/ports"
+)
+
+const (
+	queryInsertMenu              = `INSERT INTO food_menu (name,thumbnail,price,is_active,user_id) values (?,?,?,?,?)`
+	queryGetMenuByUserId         = `SELECT id,name,price,thumbnail,is_active from food_menu where user_id = ?`
+	queryDeleteBatchMenuByUserId = `DELETE from food_menu where user_id = ?`
```

Comment:
Do not delete the data, you can create new column deleted_at and then if then we can set the menu to inactive and insert data deleted_at